### DISCUSSION TITLE
"Proposal for New Features: Multiple Formats, Auto-fetch User Info, Share Button, and More for Resume Generator Website"

### DISCUSSION THREAD

**Discussion by Khizarshah01** [y4sssh/Resume-Generator#28] [Ideas]
Hello everyone,

I'm using the resume generator website and I've been thinking about some new features that could improve the website's functionality and user experience. I believe these features would greatly benefit users and enhance the overall quality of the website.

Here are the features I'm proposing:

Multiple Format (Priority, Important): Allow users to choose from multiple resume formats that prioritize different sections and information.

Implement feature to automatically fetch user information from various profiles: Enable users to quickly import their information from social media, professional networking sites, and other online profiles.

Share button (Like download): Allow users to share their resumes on social media platforms or via email with a single click.

Dynamic contributors: Allow contributors to make changes to the website in real-time.

Contact Page Style: Redesign the contact page to make it more user-friendly and appealing.

Footer: Add a footer to the website that includes useful links, copyright information, and other relevant details.

Theme option: Allow users to choose from multiple theme options to customize the look and feel of their resume.

Home Page: Redesign the homepage to make it more engaging and informative for users.

Donation Page: Add a donation page to the website to enable users to support the project.

README update (Idea: Show project roadmap): Update the README file to include a roadmap that outlines the future plans for the project.

Token for investment: Enable investors to purchase tokens that represent a stake in the project.

Resume Study (page to study various successful templates): Add a page to the website that provides users with access to successful resume templates to study and learn from.

Github Logo on Website (like reactjs): Add the GitHub logo to the website to make it easier for users to find and contribute to the project.

I'd love to hear your thoughts on these proposals and whether you think they would be valuable additions to the resume generator website. Please feel free to share your feedback, suggestions, or concerns.

Thank you for your time and consideration.

**Comment by y4sssh** [comment]
What's up guys

**Comment by ShashwatAgrawal20** [comment]
- [x] Multiple Format (Priority, Important)
- [ ] Implement a feature to automatically fetch user information from various profiles
- [x] Share button ( Like download )
- [x] Dynamic contributors
- [x] Contact Page Style
- [x] Footer  
- [x] Theme option
- [x] Home Page
- [ ] Donation Page
- [x] README update ( Idea: Show project road map )
- [ ] Token for investment
- [x] Resume Study ( page to study various successful templates )
- [x] Github Logo on Website ( like reactjs )
- [ ] Template likes button
- [ ] feedback option
- [x] Resume Study Page

**Reply by Khizarshah01** [reply]
good job @ShashwatAgrawal20
